spec/queuesTest.js

```diff
@@ -0,0 +1,65 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import Queues from '../src/queues'
+
+chai.use(chaiChange)
+
+describe.only('Queues', () => {
+  'use strict'
+
+  it('exists', () => {
+    expect(Queues).to.be.a('function')
+  })
+
+
```

Comment:
Check your white spacing format so as to keep a consistent look and feel for readability.